dawn.json

```diff
@@ -997,6 +1004,7 @@
         ]
     },
     "texture builder": {
+        "_comment": "This builder is kept for testing only",
```

Comment:
After your patch there is no more mention of the texture builder so this isn't needed. We kept it for buffers because it was the builder being used to test the "wire" implementation of the builder error callback. This builder isn't used like that so it can be safely removed.